git-hub

```diff
@@ -96,76 +123,91 @@ def push():
     "git push user pr/num:branch"
 
     """
+    p = subprocess.Popen("git branch", stdout=subprocess.PIPE)
+    pr = str(p.stdout.read())
+    pr = pr.split("*", 1)[1].split()[0]
+    pr = pr.replace("\\n", "").replace(" ", "").replace("'", "").split("/")[1]
     try:
-        with open(".git_hub_cache") as json_file:
-            data = json.load(json_file)
-            other_user = data["user"]
-            pr = data["pr"]
-            branch = data["branch"]
-            p = subprocess.Popen(["git", "push", other_user, f'{pr}:{branch}'], stdout=subprocess.PIPE)
-            p.stdout.read()
-    except FileNotFoundError as e:
-        print(e)
-        print("This command must be used after git hub pr.")
-        sys.exit(1)
+        path_prs = path_to_toml()
+        f = open(path_prs, "r")
+        pr_dict = toml.load(f)      # fetches toml file and creates a dictionary
+    except (OSError, IOError) as e:
+        # if pull-requests.toml hasnt been created yet calls sync and then reties to fetch
+        sync()
+        push()
 
+    user = pr_dict['open pull requests'][pr]['user']
+    branch = pr_dict['open pull requests'][pr]["branch"]
+    p = subprocess.Popen(["git", "push", user,
+                         f'pr/{pr}:{branch}'], stdout=subprocess.PIPE)
+    p.stdout.read()
 
-def find_path_to_git():
+
+def path_to_git():
+    """Finds path to .git folder
+    """
     path_repo = os.path.abspath('.')
-    while os.path.abspath(path_repo) != '/' and not os.path.isdir(pjoin(path_repo, '.git')):  # if not in directory with .git, keep going back to find file
+    # if not in directory with .git, keep going back to find file
+    while os.path.abspath(path_repo) != '/' and not os.path.isdir(pjoin(path_repo, '.git')):
         path_repo = pjoin(path_repo, '..')
     path_git = pjoin(path_repo, ".git")
     return path_git
 
 
-def find_in_dictionary(pr_num, open_or_closed_dictionary):
+def path_to_toml():
+    """Finds path to pull-requests.toml
+    """
+    path_git = path_to_git()
+    path_github = pjoin(path_git, 'git-hub')
+    path_prs = pjoin(path_github, 'pull-requests.toml')
+    return path_prs
+
+
+def find_in_dictionary(pr_data, open_or_closed_dictionary):
     """Fetches all the information on a certain PR
 
     Parameters
     ----------
-    pr_num : integer
+    Pr_data : integer
         number of PR to fetch information from
-    open_or_closed_dictionary : dictionary
+    Open_or_closed_dictionary : dictionary
```

Comment:
parameters should not be capitalized; `pr_data`.  Typically, we also do not specify the type of the object as is done here.  What is stored in this variable; can we come up with a more descriptive name?

How about `pr_id` and `pr_data` for the two?